doc/FEATURES_RELEASE_2.md

```diff
@@ -53,22 +54,136 @@ must display useful error messages about such constructs to the user.
+sequential DDI and its enclosing construct is the task on line 2.
+
+## Identify Parallel DDIs - Complete
+
+### Description
+Parallel DDIs arise because the PML contains interacting drugs in two different
+"arms" of a PML branch construct. The system should identify these parallel DDIs
+and the enclosing `branch` construct.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/parallel.pml` which contains
+a parallel DDI. Press `Submit`. The system should display the DDI information
+for the parallel DDI found in the analysis tab including its enclosing construct
+and the drugs involved.
+
+Specifically, it should show that the torasemide/trandolapril interaction is a
+parallel DDI and its enclosing construct is the branch on line 2.
+
+## Report Alternative non-DDIS - Complete (Bonus)
+
+### Description
+Alternative non-DDIs arise because the PML contains interacting drugs in
+different "arms" of a PML selection construct. These are not DDIs but have the
+potential to become DDIs if contained within an iteration construct. The system
+should identify these alternative non-DDIs and the enclosing `selection`
+construct. It should display them as a warning to signal potential DDIs if not
+carefully handled.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/alternative_non_ddi.pml` which
+contains an alternative non-DDI. Press `Submit`. The system should display the
+information for the alternative non-DDI found in the analysis tab including its
+enclosing construct and the drugs involved.
+
+Specifically, it should show that the torasemide/trandolapril interaction is an
+alternative non-DDI and its enclosing construct is the selection on line 2.
+
+## Report Repeated Alternative DDIs - Complete (Bonus)
+
+### Description
+Repeated alternative DDIs arise because the PML contains interacting drugs in
+different "arms" of a PML selection construct. However this selection occurs
+within an iteration which may lead to an interaction when the selection is
+repeated and another "arm" is chosen. The system should identify these repeated
+alternative DDIs and the enclosing `selection` and `iteration` constructs.
+
+### Testing
+Visit the [homepage]. Select `panacea/test/fixtures/repeated_alternative.pml`
+which contains a repeated alternative DDI. Press `Submit`. The system should
+display the information for the repeated alternative DDI found in the analysis
+tab including its enclosing construct and the drugs involved.
+
+Specifically, it should show that the torasemide/trandolapril interaction is a
+repeated alternative DDI and its enclosing constructs are the selection on line
+3 and the iteration on line 2.
+
+## Specify a Delay - Complete
+
+### Description
+The system should provide a way to say, at any point in a sequential workflow,
+that the enactment of the pathway will pause/delay for a given time.
+
+Users can specify a delay using the following construct:
+
+`time { ...list of time values }`
```

Comment:
Listing the valid time ranges for each value is worth doing too. We should also mention that order is irrelevant.